### ISSUE TITLE
error in score

### ISSUE THREAD

**Issue by ghost** [ryanb/govsgo#36]
Example in this game : http://govsgo.com/games/2963
You say black won by 48.5, but White won by 3.5 according to cgoban3 score util.

**Comment by ryanb** [comment]
Thanks for reporting this. GNU Go is what handles the scoring and it appears to consider the white group at C14 to be dead. I don't see two eyes there so in this case I think GNU Go is correct in its scoring.

While not the case here, sometimes GNU Go does not handle Seki properly and will incorrectly mark one of the groups dead. I will leave this ticket open because it should be possible for the user to mark a group as alive or dead in those situations.

**Comment by ghost** [comment]
Oh he, haha, yes, that's a false eye. Sorry.

So another improvement is needed, to see at a glance what groups gnugo consider dead.

**Comment by ryanb** [comment]
@lorill, yeah, I agree. I don't think GNU Go supports this, but I plan to re-do the territory/scoring at some point which would make this possible.
